mod-source-record-storage-server/src/main/java/org/folio/services/util/AdditionalFieldsUtil.java

```diff
@@ -15,11 +21,7 @@
 import org.marc4j.MarcReader;
 import org.marc4j.MarcStreamWriter;
 import org.marc4j.MarcWriter;
-import org.marc4j.marc.ControlField;
-import org.marc4j.marc.DataField;
-import org.marc4j.marc.MarcFactory;
-import org.marc4j.marc.Subfield;
-import org.marc4j.marc.VariableField;
+import org.marc4j.marc.*;
```

Comment:
@KyryloShulzhenko, could you please remove "*"? Just set a huge value in imports in intellijidea.